R/install-dev.sh

```diff
@@ -38,7 +38,12 @@ pushd $FWDIR > /dev/null
 if [ ! -z "$R_HOME" ]
   then
     R_SCRIPT_PATH="$R_HOME/bin"
-   else
+  else
+    # if command 'which R' finds no R home, then exit
+    if ! which R >/dev/null; then
```

Comment:
I think "command" is the preferred way to do this. See how java is detected in `spark-class`.